src/main/java/com/unb/tracker/TrackerController.java

```diff
@@ -21,12 +21,23 @@ public String index(ModelMap map) {
     }
 
     @Autowired
+    private CourseRepository courseRepository;
+    @RequestMapping(value="/addCourse", method = RequestMethod.GET, produces = {MediaType.TEXT_HTML_VALUE})
+
+    public @ResponseBody
+    String addNewCourse (@RequestParam String name) {
+        Course n = new Course();
+        n.setName(name);
+        courseRepository.save(n);
+        return "Saved";
+    }
+
+    @Autowired
     private UserRepository userRepository;
-    @RequestMapping(value="/add", method = RequestMethod.GET, produces = {MediaType.TEXT_HTML_VALUE})
+    @RequestMapping(value="/addUser", method = RequestMethod.GET, produces = {MediaType.TEXT_HTML_VALUE})
```

Comment:
lowecase URL path

EDIT: I realize this is just my convention, sorry! We should discuss this and follow a convention

Reply:
I'm working on the URL path now :+1: